pkgs/development/python-modules/sphinx-sitemap/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, sphinx
+, pytest
```

Comment:
Use `pytestCheckHook` instead and drop the explicit `checkPhase`.

If you have to pass arguments to `pytest`, use `pytestFlagsArray`.